# VIC
VIC is a bioinformatics software tool for the interpretation of sequence variants in cancer by the AMP-ASCO-CAP 2017 standards and guidelines. The input to VIC is an annotated file generated by ANNOVAR or un-annotated AVinput/VCF files, while the output of VIC is the classification of variants into 'Benign', 'Likely benign', 'Uncertain significance', 'Likely pathogenic' and 'Pathogenic', together with detailed evidence scores.
To see the help menu, use the program as "java -jar VIC/dist/VIC.jar --help":

    Notice: Your command of VIC is [--help]
    usage: java -jar VIC.jar [-b <arg>] [-d <arg>] [-h] [-input_type <arg>]
         [-i <arg>] [-cancer_type <arg>] [-convert2annovar <arg>] [-l <arg>]
         [-skip_annovar] [-o <arg>] [-annotate_variation <arg>] [-table_annovar
         <arg>] [-s <arg>] [-otherinfo <arg>] [-db <arg>]
    -annotate_variation <arg>           <path>The Annovar perl script of
                                       annotate_variation.pl
    -b,--buildver <arg>                 The genomic build version, it can be
                                       hg19 and will support GRCh37 hg18 GRCh38 later
    -cancer_type <arg>                  The cancer type, please check the doc
                                       for the details of cancer type
    -convert2annovar <arg>              <path>The Annovar perl script of
                                       convert2annovar.pl
    -d,--database_locat/humandb <arg>   <path>The  database location/dir for
                                       the annotation dataset
    -db,--database_vic <arg>            --required argument: <path>The
                                     database location/dir for the CancerVar dataset file
    -h,--help                           help message
    -i,--inputfile <arg>                --required argument: The input file
                                     contains your variant
    -input_type <arg>                   --required argument: The input file
                                     type, it can be  AVinput(Annovar's format),VCF(VCF with single
                                     sample),VCF_m(VCF with multiple samples)
    -l,--user_specified_list <arg>      User specified variant list with
                                     self-designated clinical impact significance, must be in AVinput format
    -o,--outputfile <arg>               --required argument: The prefix of
                                     output file which contains the results, the file of results will be as
                                     [$$prefix].cancervar
    -otherinfo <arg>                    true or false:print out otherinfo
                                     (infomration in fifth column in queryfile,default: TRUE). this option only
                                     perform well with AVinput file,and the other information only can be put
                                     in the fifth column.  The information in >5th column will be lost. When
                                     input as  VCF or VCF_m files with otherinfo option, only het/hom will be
                                     kept, depth and qual will be lost, the cancer type should be provide by
                                     command option.
    -s,--evidence_file <arg>            User specified Evidence file for each
                                     variant
    -skip_annovar                       Skip the Annovar annotation, this can
                                     be true only after you  already got the annovar's annotation result
    -table_annovar <arg>                <path>The Annovar perl script of
                                     table_annovar.pl
    ####################################################################################################################
    Example:
    java -jar VIC.jar -annotate_variation /path/to/annotate_variation.pl -b hg19 -convert2annovar /path/to/convert2annovar.pl 
    -table_annovar /path/to/table_annovar.pl -d /path/to/annovar/humandb -db /path/to/vicdb -i inputfile.avinput 
    -input_type AVinput -o outputfile 
